### PR TITLE
Update deps

### PR DESCRIPTION
**Pull request by hkalodner**
- Update NPM dependencies
- Fix typescript compilation

### CODE REVIEW

File: docs/Inside_Arbitrum.md [hunk 1]
```diff
@@ -206,38 +204,38 @@ Here’s another example of what the chain state might look like, if several val
...
-  * A special case of adding your stake to a successor block is when you create a new rollup block as a successor to a block you’re already staked on.
-* If you’re staked only on the latest confirmed block (and possibly earlier blocks), you or anyone else can ask to have your stake refunded. Your staked funds will be returned to you, and you will no longer be a staker. 
-* If you lose a challenge, your stake is removed from all blocks and you forfeit your staked funds.
+- If you’re not staked, you can stake on the latest confirmed rollup block. When doing this, you deposit with the EthBridge the current minimum stake amount.
+- If you’re staked on a rollup block, you can also add your stake to any one successor of that block. (The EthBridge tracks the maximum rollup block number you’re staked on, and lets you add your stake to any successor of that block, updating your maximum to that successor.) This doesn’t require you to place a new stake.
+  - A special case of adding your stake to a successor block is when you create a new rollup block as a successor to a block you’re already staked on.
```
Comment (joshuacolvin0) [inline docs/Inside_Arbitrum.md:236]: extra spaces?

File: docs/Inside_Arbitrum.md [hunk 2]
```diff
@@ -251,18 +249,18 @@ The rules for resolving rollup blocks are fairly simple.
...
-* the block’s predecessor has been rejected, or
-* all of the following are true:
-  * the block’s deadline has passed, and
-  * there is at least one staker, and
-  * no staker is staked on the block.
+- the block’s predecessor has been rejected, or
+- all of the following are true:
+  - the block’s deadline has passed, and
+  - there is at least one staker, and
+  - no staker is staked on the block.
```
Comment (joshuacolvin0) [inline docs/Inside_Arbitrum.md:263]: more alignment issues

**Review by joshuacolvin0 — DISMISSED** [review]
LGTM

File: docs/Inside_Arbitrum.md (hunk 2 above)
Comment (joshuacolvin0) [reply docs/Inside_Arbitrum.md:263]: actually on purpose indented

File: docs/Inside_Arbitrum.md (hunk 1 above)
Comment (joshuacolvin0) [reply docs/Inside_Arbitrum.md:236]: nevermind, indention on purpose

**Review by PlasmaPower — DISMISSED** [review]
LGTM

**Review by DZGoldman — APPROVED** [review]

**Pull request merged**